test/python/test_pack.py

```diff
@@ -555,22 +554,6 @@ def test_project_without_build_dockerfile(cartridge_cmd, project_without_depende
     assert process.returncode == 0
 
 
-@pytest.mark.skip()
```

Comment:
Why do you drop tests?

Reply:
This test duplicates the same one from `test_pack_docker`